modules/ROOT/pages/docker/operations.adoc

```diff
@@ -337,3 +313,53 @@ For example, to install the APOC plugin (`apoc`) and the Neo Semantics plugin (`
 ====
 Running Bloom in a Docker container requires Neo4j Docker image 4.2.3-enterprise or later.
 ====
+
+If invalid `NEO4J_PLUGINS` values are passed, Neo4j returns a notification that the plugin is not known.
+For example, `--env NEO4J_PLUGINS='["gds"]'` returns the following notification:
+
+.Example output
+[source, shell, role="noheader"]
+----
+"gds" is not a known Neo4j plugin. Options are:
+apoc
+apoc-core
+bloom
+graph-data-science
+graphql
+n10s
+----
+
+.Install the APOC plugin (`apoc`)
+====
+You can use the Docker argument `--env NEO4J_PLUGINS='["apoc"]'` and run the following command:
+
+[source, shell, subs="attributes"]
+----
+docker run -it --rm \
+  --publish=7474:7474 --publish=7687:7687 \
+  --user="$(id -u):$(id -g)" \
+  -e NEO4J_AUTH=none \
+  --env NEO4J_PLUGINS='["apoc"]' \
+  neo4j:{neo4j-version-exact}
+----
+====
+
+.Install the APOC plugin (`apoc`) and the Neo Semantics plugin (`n10s`)
+====
+You can use the Docker argument `--env NEO4J_PLUGINS='["apoc", "n10s"]'` and run the following command:
+
+[source, shell, subs="attributes"]
+----
+docker run -it --rm \
+  --publish=7474:7474 --publish=7687:7687 \
+  --user="$(id -u):$(id -g)" \
+  -e NEO4J_AUTH=none \
+  --env NEO4J_PLUGINS='["apoc", "n10s"]'
+  neo4j:{neo4j-version-exact}
+----
+====
+
+[NOTE]
+====
+The `--user="$(id -u):$(id -g)"` argument is required to ensure that the plugins are installed with the correct permissions.
+====
```

Comment:
this is not true, you can delete the `--user="$(id -u):$(id -g)"` from all the examples and they should still work.

Reply:
yes I think it's a good idea to remove the line from the examples.